.github/workflows/deploy.yml

```diff
@@ -1,16 +1,27 @@
-name: Deploy to dev
+name: Deploy
 
 on:
+  workflow_call: # Called after tests pass on master branch push
+    inputs:
+      environment:
+        description: 'Environment'
+        type: string # environment type not supported here, yet
+        default: dev
   workflow_dispatch:
-  push:
-    branches: [master]
+    inputs:
+      environment:
+        description: 'Environment'
+        type: environment
+        required: true
+        default: dev
 
```

Comment:
Hmm, just thinking about TRC's use case a bit; we'd love to be able to trigger OTP prod to deploy when we publish a new GTFS. I *think*, now that you're putting this in here (thanks! 🙏 ), we can actually do that via [`repository_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch). That is, you can have here:

```yaml
on:
  repository_dispatch:
    types: [trigger_deploy]
```

and then we, over in `gtfs_creator`, can trigger it by POST'ing to `api.github.com` with the right incantation. (I saw [this comment](https://github.community/t/triggering-by-other-repository/16163) with the idea.)

Would you be up for adding that to the workflow here, so we could try it out a bit on our end? I think we could provide the `environment` in our JSON post, and so update dev while we test.

Reply:
That sounds really snazzy! Happy to add that here.